language/tools/move-cli/src/base/test.rs

```diff
@@ -60,6 +60,11 @@ pub struct Test {
     /// Show the storage state at the end of execution of a failing test
     #[clap(name = "global_state_on_error", short = 'g', long = "state_on_error")]
     pub report_storage_on_error: bool,
+
+    /// Ignore compiler's warning, and continue run tests
+    #[clap(name = "ignore_compile_warnings", long = "ignore_compile_warnings")]
+    pub ignore_compile_warnings: bool,
```

Comment:
This defaults to false right? 

I think it would be fine to flip this flag, I think they should be ignored by default